index.js

```diff
@@ -0,0 +1,112 @@
+document.addEventListener("DOMContentLoaded" ,() =>{
+    let getPokemon = document.querySelector("#getPokemon")
+    let battle = document.querySelector("#battle")
+    let pokemon1 = document.querySelector("#pokemon1")
+    let pokemon2 = document.querySelector("#pokemon2")
+    let audio = document.querySelector("audio")
+    let randomWinner = []   
+    const getThatPokemon = async () => {
+        try {
+            random = Math.floor(Math.random() * Math.floor(809))
+            let res = await axios.get(`http://pokeapi.co/api/v2/pokemon/${random}/`)  
+            let pokemonName = res.data.name
+            let h1 = document.createElement("h1")
+            h1.innerText = pokemonName
+            pokemon1.appendChild(h1)
+            randomWinner.push(pokemonName)
+            
+            let image = document.createElement("img")
+            image.src = res.data.sprites.back_default
+            pokemon1.appendChild(image)
+
+            let h2 = document.createElement("h2")
+            h2.innerText = `HP: ${res.data.stats[5].base_stat}`
+            pokemon1.appendChild(h2)
+            
+            let res2 = await axios.get(res.data.moves[0].move.url)
+            let res3 = await axios.get(res.data.moves[1].move.url)
+            let res4 = await axios.get(res.data.moves[2].move.url)
+            let res5 = await axios.get(res.data.moves[3].move.url)          
```

Comment:
This is hard-coding. Imagine if you had 40 moves to get through, you don't want to repeat the same code line by line. You can just use a for loop and loop through the array of moves and for each of them get the name, the url, get pp from that url, create element, and append it. Keep your code DRY :)